src/base/emu-i386/simx86/interp.c

```diff
@@ -299,16 +291,16 @@ static unsigned int _JumpGen(unsigned int P2, int mode, int cond,
 #endif
 		}
 		if (CONFIG_CPUSIM)
```

Comment:
By the way, this condition can probably
be removed. As these things are accessed
by va_args, I don't think an extra arg will
make a problem for SIM.